src/components/atoms/Egg/index.jsx

```diff
@@ -0,0 +1,87 @@
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBIcon,
+  MDBPopover,
+  MDBPopoverHeader,
+  MDBPopoverBody,
+  MDBAnimation,
+} from "mdbreact";
+
+class Egg extends React.Component {
+  state = {};
+
+  componentDidMount = () => {
+    const res = localStorage.getItem("egg_" + this.props.id);
+
+    if (res) {
+      this.setState({
+        found: true,
+      });
+    } else {
+      this.setState({
+        found: false,
+      });
+    }
+  };
+
+  setEgg = () => {
+    if (!this.state.found) {
+      this.setState(
```

Comment:
You should change this to: 
this.setState({
        found: true,
        set: true,
},

Reply:
Wrong. Prettier tells you otherwise.